src/backend/distributed/commands/multi_copy.c

```diff
@@ -3188,6 +3324,37 @@ InitializeCopyShardState(CopyShardState *shardState,
 }
 
 
+static void
+CloneCopyOutStateForLocalCopy(CopyOutState from, CopyOutState to)
```

Comment:
missing function comment.

```c
/*
 * CloneCopyOutStateForLocalCopy creates a shallow copy of the CopyOutState with a new
 * fe_msgbuf. We keep a separate CopyOutState for every local shard placement, because
 * in case of local copy we serialize and buffer incoming tuples into fe_msgbuf for each
 * placement and the serialization functions take a CopyOutState as a parameter.
 */
```